packages/design-system/src/components/OcButton/OcButton.vue

```diff
@@ -32,6 +33,20 @@ export default defineComponent({
         return ['button', 'a', 'router-link'].includes(value)
       }
     },
+
+    /**
+     * HTML-Type of the button
+     * submit: Submit a form (@submit)
+     * reset: Reverts any changes in a form (@reset)
+     * button: Default. Defines a clickable button, click event needs to be catured manually
+     */
+    buttonType: {
+      type: String,
+      default: 'button',
+      validator: (value: any) => {
+        return ['button', 'submit', 'reset'].includes(value)
+      }
+    },
```

Comment:
We  already have the submit prop for this, it's a bit awkward namewise but it should work
If you insist on renaming it (I'd rather not) please keep the handling via additionalAttributes which only adds the prop to type button elements